client/components/ConditionCheckbox/ConditionCheckbox.js

```diff
@@ -0,0 +1,54 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Checkbox from 'components/Checkbox/Checkbox';
+import styles from './ConditionCheckbox.css';
+
+export default class ConditionCheckbox extends PureComponent {
+  constructor() {
+    super();
+    this.state = {
+      allChecked: false,
+      checkedCount: 0,
+      arr: [],
+      options: [
+        { value: '1', text: 'Affordable Class' },
+        { value: '2', text: 'Middle Class' },
+        { value: '3', text: 'High Class' },
+        { value: '4', text: 'Luxurious Class' },
+      ],
+    };
+  }
+
+  static propTypes = {
+    price: PropTypes.object,
+    action: PropTypes.func,
+  };
+
+  handleOnBlurAction = (e) => {
+    const arr = this.state.arr;
+    if (e.target.checked) {
+      if (arr.indexOf(e.target.value) === -1) {
```

Comment:
if you are using e.target.value so many times, assign a const to it?

Reply:
Yes it should. Sorry.  Forgot todo it.